cmd/manager/common.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
```

Comment:
nit: import ordering

Reply:
this was generated by VSCode's import ordering linter :/